composer/trainer/dist_strategy.py

```diff
@@ -181,18 +181,18 @@ def prepare_fsdp_module(model: torch.nn.Module, optimizers: Optional[Union[torch
         raise ValueError('FSDP CPU Offload not supported yet.')
 
     mixed_precision = fsdp_config.get('mixed_precision', 'DEFAULT')
+    param_dtype = None
+    reduce_dtype = None
+    buffer_dtype = None
     if isinstance(mixed_precision, dict):
-        param_dtype = get_torch_dtype(mixed_precision.get('param_dtype', 'float32'))
-        reduce_dtype = get_torch_dtype(mixed_precision.get('reduce_dtype', 'float32'))
-        buffer_dtype = get_torch_dtype(mixed_precision.get('buffer_dtype', 'float32'))
+        param_dtype = get_torch_dtype(mixed_precision.get('param_dtype', None))
+        reduce_dtype = get_torch_dtype(mixed_precision.get('reduce_dtype', None))
+        buffer_dtype = get_torch_dtype(mixed_precision.get('buffer_dtype', None))
```

Comment:
Also passing `None` into `get_torch_dtype` will throw an error.
either need to update `get_torch_dtype` or do something like
```
param_dtype = mixed_precision.get('param_dtype', None)
if param_dtype is not None:
    param_dtype = get_torch_dtype(param_dtype)
```